var/spack/repos/builtin/packages/nfs-utils/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class NfsUtils(AutotoolsPackage):
+    """FThe NFS Utilities package contains the userspace server and client tools
```

Comment:
```suggestion
    """The NFS Utilities package contains the userspace server and client tools
```

Reply:
Fix it in https://github.com/spack/spack/pull/15491/commits/c502945e31f42c78487f9299d734288553160bb5